### PR TITLE
[G7T5-49] Calling API to View Course Status

### PR DESCRIPTION
**Pull request by branfats**
# Description

Fixes [Jira ticket #49](https://is212g7t5.atlassian.net/browse/G7T5-49?atlOrigin=eyJpIjoiMWViNzg2ZDljNWEwNGVhZGE4NDJlMjhmMGVjOWUzNjMiLCJwIjoiaiJ9)

Display Course status in Courses page when logged in as "Staff"
Screenshot 2022-10-20 at 4 11 06 PM

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
- [ ] I have added the corresponding testcases into the [testcase document](https://docs.google.com/spreadsheets/d/1QOyUN_kFN0fddLkjAQz2DK3jou3cWdN9CJsNbl3v0Kg/edit#gid=0)

Please indicate the testcase ID you have added or are using

- [ ] G7T5-1-1
- [ ] G7T5-1-2

# Checklist:

- [x] I have added the appropriate labels to my PR
- [x] My code follows the style guidelines of this project
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/src/components/course/CourseStatusBadge.jsx [hunk 1]
```diff
@@ -1,23 +1,50 @@
...
 
-export default function CourseStatusBadge({registrationStatus, completionStatus}) {
+export default function CourseStatusBadge({ staffId, courseId }) {
   const statusToBadgeClass = {
-    "Completed": "bg-green-100 text-green-800",
-    "Rejected": "bg-red-100 text-red-800", 
-    "Waitlisted": "bg-blue-200 text-blue-800",
-    "Registered": "bg-yellow-100 text-yellow-800",
-    "Not Registered": "bg-purple-100 text-purple-800"
+    Completed: "bg-green-100 text-green-800",
```
Comment (charmjj) [inline app/src/components/course/CourseStatusBadge.jsx:6]: rmb the inverted commas around your keys

File: app/src/components/course/CourseStatusBadge.jsx [hunk 2]
```diff
@@ -1,23 +1,50 @@
...
+            ? registrationData.completionStatus.trim()
+            : registrationData.regStatus.trim(),
+        );
+      }
+    }
+
+    getCourseStatus(staffId, courseId);
+  }, []);
+
+  if (courseStatus === "Completed") {
```
Comment (charmjj) [inline app/src/components/course/CourseStatusBadge.jsx:33]: dont think u need this if-block here

File: app/src/components/course/CourseStatusBadge.jsx (hunk 1 above)
Comment (branfats) [reply app/src/components/course/CourseStatusBadge.jsx:6]: Oops, will make changes thanks!

**Review by jonathantan1425 — APPROVED** [review]
if tested

**Pull request merged**